charts/karpenter/values.yaml

```diff
@@ -52,7 +52,7 @@ terminationGracePeriodSeconds:
 # This is required when using a custom CNI.
 hostNetwork: false
 # -- Configure the DNS Policy for the pod
-dnsPolicy: ClusterFirst
+dnsPolicy: Default
```

Comment:
Wouldn't this be preferred to be overwritten from the fargate examples `values.yaml` or a `set` rather than change it for everyone?

Reply:
Karpenter doesn't have a need to reach out to coreDNS since it's not interacting with other services and there are no services that are interacting with Karpenter directly. With this in mind, I believe this is fine to take as the default

Reply:
I think this is generally applicable to all Karpenter installs since Karpenter doesn't rely on cluster internal dns lookups from core-dns.